serpent-moltres/fuel-column/README.md

```diff
@@ -1,6 +1,54 @@
+# fuel-column
+
+This section compares Serpent and Moltres results of a fuel column.
+The file `postprocessing.py` produced figures of Serpent and Moltres results.
+
+Serpent input files:
+--------------------
+* standard-column-LBP: column of standard fuel assemblies with bottom and top reflectors. It includes lumped burnable poisons in the corners of the hexagon.
```

Comment:
Best practice limits line lengths to 80 chars. Please make this a setting in your text editor. 
```suggestion
* standard-column-LBP: column of standard fuel assemblies with bottom and top reflectors. 
It includes lumped burnable poisons in the corners of the hexagon.
```

Reply:
(I won't make comments everywhere else this happens. Please hardwrap lines through this whole PR.)